Library/Formula/clang-omp.rb

```diff
@@ -0,0 +1,62 @@
+class ClangOmp < Formula
+  homepage "https://clang-omp.github.io/"
+  url "https://github.com/clang-omp/llvm/archive/2015-04-01.tar.gz"
+  sha256 "37f990ad99b3213507ec88f86702c5a057ce397cc16638eeee5c88906572daec"
+
+  depends_on "libiomp"
+  depends_on "cmake" => :build
+
+  resource "compiler-rt" do
+    url "https://github.com/clang-omp/compiler-rt/archive/2015-04-01.tar.gz"
+    sha256 "5a8d39ff6ce524e23fae32870f85b18d43f2795da2011d3cbb6b29d471bb27b7"
+  end
+
+  resource "clang" do
+    url "https://github.com/clang-omp/clang/archive/2015-04-01.tar.gz"
+    sha256 "2717115e5ba491e3b8119311f0d792420ba41be34a89733b9880eb3d3c09fbe5"
+  end
+
+  needs :cxx11
+
+  def install
+    resource("compiler-rt").stage { (buildpath/"projects/compiler-rt").install Dir["*"] }
+    resource("clang").stage { (buildpath/"tools/clang").install Dir["*"] }
+
+    system "./configure", "--prefix=#{prefix}", "--enable-cxx11", "--enable-libcpp"
+    system "make", "install"
+
+    mv bin/"clang", bin/"clang-omp"
```

Comment:
The box it's running on has 16GB of memory so it seems unlikely that it is that.